addons/website/static/tests/tours/page_manager.js

```diff
@@ -29,6 +29,27 @@ const checkKanbanGroupBy = [{
     trigger: '.o_cp_searchview .o_facet_remove',
 }];
 
+const checkWebsiteFilter = [
+    {
+        content: "Click on My Website",
+        trigger: "button.dropdown-toggle:contains('My Website')",
```

Comment:
```suggestion
const checkWebsiteFilter = [{
    content: "Click on My Website",
    trigger: "button.dropdown-toggle:contains('My Website')",
```
Otherwise, you are losing a whole indent level for no reason, and it's already hard sometimes to make lines fit in a reasonnable length, might as well not waste 4 char.

Nitpicking but good to know :)
It's also why we never indent the first level in xml file, eg in this file: 
https://github.com/odoo/odoo/blob/e6a688815ff8ee31c5f3fcd8dbf09fd527aedd5d/addons/website_sale/views/product_product_add.xml